pyiron_atomistics/vasp/base.py

```diff
@@ -2017,6 +2017,7 @@ def collect(self, directory=os.getcwd(), sorted_indices=None):
                 raise VaspCollectError("Error in parsing OUTCAR")
             log_dict["energy_tot"] = self.outcar.parse_dict["energies"]
             log_dict["temperature"] = self.outcar.parse_dict["temperatures"]
+            log_dict["stresses"] = self.outcar.parse_dict["stresses"]
```

Comment:
Since this is inside an `elif` statement, this segment of the code would work only when vasprun files are absent. You would need to put this in if statement block:

https://github.com/pyiron/pyiron_atomistics/blob/d1d3ed1cb7657633c5a67703f5b32f4a8d8ef200/pyiron_atomistics/vasp/base.py#L1947